nextflow_schema.json

```diff
@@ -233,6 +229,10 @@
         "igenomes_ignore": {
             "type": "boolean",
             "default": false
+        },
+        "round_DE": {
+            "type": "string",
+            "default": null
```

Comment:
Why were all default values removed, but here not?

Reply:
I removed specifically the defaults for the genome params (library, keytype etc) because those can for some weird nextflow reason not be set in nextflow.config, otherwise the defaults will be used (i.e. null) and not the user params if set. For round_DE, this is not the case